handler/deployment/deployment.go

```diff
@@ -202,7 +202,7 @@ func (d *Deployment) deploymentSuccess(ctx context.Context, correlationID, evalu
 	for {
 		select {
 		case <-ctx.Done():
-			log.InfoC(correlationID, "bailing on deployment status", minLogData)
+			log.Event(ctx, "bailing on deployment status", log.INFO, minLogData)
```

Comment:
a lot of these (not all), should be
```suggestion
			log.Event(ctx, "bailing on deployment status", log.ERROR, minLogData)
```